crates/cheatnet/src/execution/syscalls.rs

```diff
@@ -297,6 +298,17 @@ impl CheatableSyscallHandler<'_> {
             // Increment, since the selector was peeked into before
             self.syscall_handler.syscall_ptr += 1;
             return self.execute_syscall(vm, deploy_syscall, constants::DEPLOY_GAS_COST);
+        } else if SyscallSelector::EmitEvent == selector {
```

Comment:
Should we increment the pointer here too?
We do it in every other case

Reply:
We should not since `self.syscall_handler.execute_next_syscall` reads the selector and increments `syscall_ptr` by itself.
Moreover, adding `syscall_ptr` incrementation makes cheatnet spy_events tests fail.

Reply:
Added a comment to the code with clarification